Experimental/DistributedAuthoritySample/Assets/Scripts/Player/AvatarTransform.cs

```diff
@@ -0,0 +1,272 @@
+using Unity.Netcode.Components;
+using Unity.Netcode.Editor;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using com.unity.multiplayer.samples.distributed_authority.input;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace com.unity.multiplayer.samples.distributed_authority.gameplay
+{
+#if UNITY_EDITOR
```

Comment:
instead of wrapping everything with #IF UNITY_EDITOR, this class should be moved to an editor-only ASMDEF file that references the runtime one and has access to its internal classes/types through the `AssemblyInfo.cs` mechanism

Reply:
Good call. I've moved this bit of code and used a custom editor using newer API.
I will, though, add namespaces in a subsequent PR. I'll create a ticket and link it here.